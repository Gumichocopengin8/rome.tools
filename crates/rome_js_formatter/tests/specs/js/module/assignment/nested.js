// Fits
bifornCringerMoshedPerplex = bifornCringerMoshedPerplexSawder = arrayO = "test"

// Exceeding:
// One assignment
bifornCringerMoshedPerplex.bifornCringerMoshedPerplexSawder.arrayOfNumber.m= "test"

// Two assignments
bifornCringerMoshedPerplex.bifornCringerMoshedPerplexSawder.arrayOfNumber = ab = "test"

// More than two assignments and exceeding: Each assignment on their own line
bifornCringerMoshedPerplex.bifornCringerMoshedPerplexSawder.array = p = ab = "tes"


bifornCringerMoshedPerplex = bifornCringerMoshedPerplexSawder = arrayOfNumb = a = "test"

tt.parenR.updateContext =    tt.braceR.updateContext =        tt.braceR.updateContext = function () {
            function a() {
                if (this.state.context.length === 1) {
                    if (this.state.context.length === 1) {
                        return;
                    }
                }
            };
        }