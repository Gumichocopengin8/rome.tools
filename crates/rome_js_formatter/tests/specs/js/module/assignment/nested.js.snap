---
source: crates/rome_js_formatter/tests/spec_test.rs
assertion_line: 223
expression: nested.js
---
# Input
// Fits
bifornCringerMoshedPerplex = bifornCringerMoshedPerplexSawder = arrayO = "test"

// Exceeding:
// One assignment
bifornCringerMoshedPerplex.bifornCringerMoshedPerplexSawder.arrayOfNumber.m= "test"

// Two assignments
bifornCringerMoshedPerplex.bifornCringerMoshedPerplexSawder.arrayOfNumber = ab = "test"

// More than two assignments and exceeding: Each assignment on their own line
bifornCringerMoshedPerplex.bifornCringerMoshedPerplexSawder.array = p = ab = "tes"


bifornCringerMoshedPerplex = bifornCringerMoshedPerplexSawder = arrayOfNumb = a = "test"

tt.parenR.updateContext =    tt.braceR.updateContext =        tt.braceR.updateContext = function () {
            function a() {
                if (this.state.context.length === 1) {
                    if (this.state.context.length === 1) {
                        return;
                    }
                }
            };
        }
=============================
# Outputs
## Output 1
-----
Indent style: Tab
Line width: 80
Quote style: Double Quotes
-----
// Fits
bifornCringerMoshedPerplex = bifornCringerMoshedPerplexSawder = arrayO = "test";

// Exceeding:
// One assignment
bifornCringerMoshedPerplex.bifornCringerMoshedPerplexSawder.arrayOfNumber.m = "test";

// Two assignments
bifornCringerMoshedPerplex.bifornCringerMoshedPerplexSawder.arrayOfNumber =
	ab = "test";

// More than two assignments and exceeding: Each assignment on their own line
bifornCringerMoshedPerplex.bifornCringerMoshedPerplexSawder.array =
	p = ab = "tes";

bifornCringerMoshedPerplex =
	bifornCringerMoshedPerplexSawder = arrayOfNumb = a = "test";

tt.parenR.updateContext =
	tt.braceR.updateContext =
		tt.braceR.updateContext = function () {
			function a() {
				if (this.state.context.length === 1) {
					if (this.state.context.length === 1) {
						return;
					}
				}
			}
		};

